### DISCUSSION TITLE
镜像开启后无法输入锁屏密码

### DISCUSSION THREAD

**Discussion by xiaokun567** [viarotel-org/escrcpy#299] [Q&A] [answered]
镜像开启后无法输入锁屏密码，是黑屏的，随便点都没反应，有root权限，能破解这个吗。

**Comment by xiaokun567** [comment, marked as answer]
直接用键盘盲输即可。哈哈

**Reply by Typhonling** [reply]
密码也许可行，PIN 码和图案就不行了。通用破法是 Disable FLAG_SECURE。[Enable Screenshot (formerly known as Disable FLAG_SECURE)](https://github.com/LSPosed/DisableFlagSecure)
